### PR TITLE
Updated generate_new_line_item

### PR DESCRIPTION
**Pull request by sulaiman-allen**

### CODE REVIEW

**Comment by chase-ramsey** [comment]
👍

File: handlers/line_item_handler.py [hunk 1]
```diff
@@ -1,80 +1,33 @@
...
+def return_report_line_items(input_file='bangazon.db'):
+    """
+    Returns a list of line items with orders that have been completed.
+    """
 
-#     # limit totals display
-#     order_sum = (order_sum if len(str(order_sum)) <= 17 else order_sum[:14] + "...") + " "
-#     customer_sum = (customer_sum if len(str(customer_sum)) <= 11 else customer_sum[:8] + "...")
-#     revenue_sum = (revenue_sum if len(str(revenue_sum)) <= 14 else revenue_sum[:11] + "...")
+    sqlite3.connet(input_file) as conn:
```
Comment (MeganWord) [inline handlers/line_item_handler.py]: missing a c in connect!

File: handlers/line_item_handler.py [hunk 2]
```diff
@@ -1,80 +1,33 @@
...
+        db = conn.cursor()
+        db.execute("SELECT li.* FROM LineItem li, Orders o WHERE li.OrderID == o.OrderID AND o.PaymentID == NULL")
```
Comment (MeganWord) [inline handlers/line_item_handler.py]: don't we want the ones where the payment id isn't null? if it's null, it's a cart, if it's a paymentID, then the order is completed... also is there a difference in sql between one equals sign and two?

**Comment by josephkane** [comment]
👍

**Pull request merged**